apps/juxtaposition-ui/src/services/juxt-web/routes/admin/admin.js

```diff
@@ -153,6 +170,41 @@ router.post('/accounts/:pid', async (req, res) => {
 			link: '/titles/2551084080/new'
 		});
 	}
+
+	let action = 'UPDATE_USER';
+	const changes = [];
+	if (oldUserSettings.account_status !== req.body.account_status) {
+		switch (req.body.account_status) {
+			case 0:
+				action = 'UNBAN';
+				break;
+			case 1:
+				action = 'LIMIT_POSTING';
+				break;
+			case 2:
+				action = 'TEMP_BAN';
+				break;
+			case 3:
+				action = 'PERMA_BAN';
+				break;
+		}
+		changes.push(`Account_status changed from "${oldUserSettings.account_status}" to "${req.body.account_status}"`);
```

Comment:
```suggestion
		changes.push(`Account status changed from "${oldUserSettings.account_status}" to "${req.body.account_status}"`);
```

Just a minor cosmetic change, but I think if we're going to the effort of writing a full messages out, then we should have "nice names" like "Account status changed" rather than "Account_status changed"

Reply:
I think we should, as the messages are designed to be human readable